packages/material-ui/src/locale/index.js

```diff
@@ -762,14 +822,18 @@ export const idID = {
 };
 
 export const isIS = {
+  localization: {
+    formatNumber: formatNumber('is-IS'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@vgaborabs Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).